cfgov/jinja2/v1/owning-a-home/closing-disclosure/index.html

```diff
@@ -0,0 +1,96 @@
+{% extends "base.html" %}
+{% import "molecules/social-media.html" as social_media with context %}
+{% set page={"search_description":"Use this tool to double-check that all the details about your loan are correct on your Closing Disclosure. If something looks incorrect or unexpected, ask why."} %}
+{% set active_page = "closing-disclosure" %}
+{% set page_js = "closing-disclosure" %}
+{% block title %}Closing Disclosure Explainer{% endblock title %}
+{% block metadescription %}Use the CFPB"s tool to double-check that all the details about your loan are correct on your Closing Disclosure.{% endblock metadescription %}
```

Comment:
`CFPB"s tool`? Needs to be an apostrophe, right?

Reply:
lol, awww dang. Fixed.